benchmarks/intel_benchmark.yml

```diff
@@ -13,6 +13,10 @@
 #  limitations under the License.
 ---
 
+- name: Update Inventory with ansible_host information
```

Comment:
Can you provide more context on this?... What type of benchmarks do you run on slurm_control_node, slurm_node, login hosts?

Reply:
We are running openmpi benchmarks for slurm